src/connection/types.ts

```diff
@@ -11,6 +13,14 @@ export enum ConnectionType {
   GNOSIS_SAFE = 'GNOSIS_SAFE',
 }
 
+export function toConnectionType(value = ''): ConnectionType | undefined {
+  if (Object.keys(ConnectionType).includes(value)) {
```

Comment:
`value in ConnectionType`?

Reply:
this is maybe a utility we could write? but there's a lot of this pattern in the codebase. I *do* like your pattern better, though.